src/main/groovy/eu/plumbr/integration/plugin/PlumbrIntegrationPlugin.groovy

```diff
@@ -0,0 +1,18 @@
+package eu.plumbr.integration.plugin
+
+import eu.plumbr.integration.artifactory.ArtifactoryIntegration
+import eu.plumbr.integration.jira.JiraIntegration
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class PlumbrIntegrationPlugin implements Plugin<Project> {
```

Comment:
We integrate our (Plumbr's) process with other services. It's like "Plumbr's Integration"